### DISCUSSION TITLE
@adonisjs/transmit setup in adonisjs version 5

### DISCUSSION THREAD

**Discussion by mohanraj-rr** [adonisjs/core#4820] [Help]
Please share some docs or suggestion, to send real-time notification to client. Using transmit in adonisjs version 5.

**Comment by RomainLanz** [comment]
Hey @mohanraj-rr! 👋🏻

Currently, there are no plans to make Transmit compatible with AdonisJS 5.

Transmit is an agnostic ESM-only library, but you might still be able to use it with AdonisJS 5. The latest version of Node.js allows importing ESM modules in CJS projects, so it’s worth giving it a try!

➡️ https://github.com/boringnode/transmit
